tests/auth_tests/test_templates.py

```diff
@@ -52,3 +42,28 @@ def test_titles(self):
         response = PasswordChangeDoneView.as_view()(request)
         self.assertContains(response, '<title>Password change successful</title>')
         self.assertContains(response, '<h1>Password change successful</h1>')
+
+
+@override_settings(ROOT_URLCONF='auth_tests.urls')
+class AuthTemplateWithClientTests(TestCase):
+
+    def setUp(self):
+        self.pr_client = PasswordResetConfirmClient()
```

Comment:
I had a similar thought though I wasn't sure if the change would be an improvement or not. "pr" me think "pull request". I haven't reviewed this in detail yet.